src/scripts/main.js

```diff
@@ -1,3 +1,53 @@
 'use strict';
 
 // write your code here
+
+const logo = document.querySelector('.logo');
+const bd = document.querySelector('body');
+
+const rejected = (resolve, reject) => {
+
+
+  function rejectHandler() {
+    const newDivError = document.createElement('div');
+    setTimeout(() => {
+
+      newDivError.className = 'error-message';
+      newDivError.innerText = 'Promise was rejected!';
+      bd.appendChild(newDivError);
+      reject(new Error('not ok'));
+    }, 3000);
+
+    setTimeout(() => {
+      newDivError.remove();
+    }, 6000);
+  }
+
+  return rejectHandler();
+};
+
+const firstPromise = new Promise(rejected);
+
+const resolver = (resolve, reject) => {
+  logo.addEventListener('click', () => {
+    function resolveHandler() {
+      const newDiv = document.createElement('div');
+
+      newDiv.className = 'message';
+      newDiv.innerText = 'Promise was resolved!';
+      bd.appendChild(newDiv);
+
+      setTimeout(() => {
+        newDiv.remove();
+      }, 3000);
```

Comment:
it should be in the then function

Reply:
have put the functions inside then & catch